src/pudl/analysis/state_demand.py

```diff
@@ -17,23 +17,12 @@
 electricity demand estimates (e.g. at the county level).
 """
 
-import datetime
-from collections.abc import Iterable
-from typing import Any
-
 import geopandas as gpd
-import numpy as np
 import pandas as pd
-from dagster import AssetOut, Field, asset, multi_asset
+from dagster import Field, asset
 
-import pudl.analysis.timeseries_cleaning
-import pudl.logging_helpers
-import pudl.output.pudltabl
```

Comment:
For some reason when you remove this `import pudl.output.pudltabl` I end up getting a circular import error, which is breaking the tests right now. `pudltabl` was already not being used here before, so it's completely extraneous but also apparently load bearing. I'll keep trying to figure this out, but I've spent awhile already debugging. @zaneselvans let me know if you have any thoughts on this weird one.

Reply:
We're going to delete `pudltabl` as soon as the data validations / tests are refactored, so I think it's not worth debugging -- just leave it in if it makes things work.